src/main/scala/jawnfs2/package.scala

```diff
@@ -17,10 +18,10 @@ package object jawnfs2 {
     * @tparam J the JSON AST to return
     * @param mode the async mode of the Jawn parser
     */
-  def parseJson[F[_], A, J](mode: AsyncParser.Mode)(implicit A: Absorbable[A], facade: RawFacade[J]): Pipe[F, A, J] = {
+  def parseJson[F[_], A, J](mode: AsyncParser.Mode)(implicit F: ApplicativeError[F, Throwable], A: Absorbable[A], facade: RawFacade[J]): Pipe[F, A, J] = {
```

Comment:
Should this have an `E >: ParseException` parameter to replace `Throwable`?  What we have is simpler and great if `F` is a `Sync` (in practice, probably), but hostile to binary effects.

Reply:
If someone wants to translate the above at an edge, they can do so. I have made it stronger in the past and in general regret the decision.